projects/go-lib/src/lib/components/go-table/go-table.component.spec.ts

```diff
@@ -149,18 +149,6 @@ describe('GoTableComponent', () => {
     });
   });
 
-  describe('afterViewInit', () => {
-    afterEach(() => {
-      component.selectAllControl.setValue(false);
-    });
-
-    it('should select all rows if preselected', () => {
-      component.tableConfig.preselected = true;
-      component.ngAfterViewInit();
-      expect(component.selectAllControl.value).toBe(true);
-    });
-  });
-
```

Comment:
Again, not really related to this PR... but we have a block for `ngAftgerViewInit` at the bottom of the page and I am consolidating.